### PR TITLE
embertest currentUrl => currentURL

### PR DESCRIPTION
**Pull request by James1x0**
Changed def, bumped version.
/ref http://emberjs.com/api/classes/Ember.Test.html#method_currentURL

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "globals",
-  "version": "8.12.0",
+  "version": "8.13.0",
```
Comment (sindresorhus) [inline package.json]: Don't bump versions in other people's projects ;)

I for one have a script for publishing which does this. You can't possible know which version I will pick either. I might want to do other changes with this one.

Comment (James1x0) [inline package.json]: :+1: Have had it both ways.

**Comment by nzakas** [comment]
:+1:

**Comment by sindresorhus** [comment]
@nzakas Would you consider this to be a breaking change and a major release?

**Comment by es128** [comment]
I'd call it a bugfix. 2c

**Comment by nzakas** [comment]
Yeah, I think it's a bug fix.

**Pull request closed without merging**